protocols/ads/src/main/resources/protocols/ads/ads.mspec

```diff
@@ -281,7 +281,8 @@
             // 4 bytes	The ADS server checks if the value changes in this time slice. The unit is 1ms
             [simple uint 32 'cycleTime']
             // 16bytes	Must be set to 0
-            [reserved   uint       128       '0x0000' ]
+            [reserved   uint       64       '0x0000' ]
```

Comment:
short question here: what was the reason for this change?

Reply:
other than that LGTM

Reply:
I think not all languages support 128bit integers

Reply:
So instead of one 128bit zero we now have two times 64bit

Reply:
golang and java do support it (BigInteger/big.Int)

Reply:
Good question. Reading my commit message again, the ParseException I got tells a different story: "Unsigned Big Integer can only contain max 64 bits"
I'll look if there is another way around that.

Reply:
I changed that because I was running into problems ... so I wasn't randomly changing things ;-) In this case I thought fixing it wans't worth the effort. I mean .... it's for writing 16 zero bytes ...

Reply:
ja let's use that then... I'll check later if we can support that (bit strange that it compiled before.)